src/settings/components/Capabilities/CapabilitiesSettings.js

```diff
@@ -24,30 +24,23 @@ const CapabilitiesSettings = ({ content, readOnly, onChangeCapabilityCheckbox })
     // policies: formatMessage(columnTranslations.policies
   };
 
+  const renderItemActionCheckBox = (item, action) => {
+    if (!readOnly && item.action !== action) return null;
```

Comment:
As above: Please explain; it is not immediately obvious to me how the conditions relate to hiding the checkbox.

Reply:
Added docstrings